docker/conf/homeserver.yaml

```diff
@@ -184,3 +184,21 @@ trusted_key_servers:
 
 password_config:
```

Comment:
Occasionally, I see spans that have a super massive duration and shows up with `213503982d 8h` in the Jaeger UI. I only see one other reference to this magic value in this issue https://github.com/jaegertracing/jaeger-ui/issues/397#issuecomment-937704585 which they describe as the "max duration".

<img width="1194" alt="" src="https://user-images.githubusercontent.com/558581/183526530-66c391a8-0fe6-4085-84d6-db24621c3469.png">

 
I can't figure out the math that makes this situation happen. Whether it be forgotten `end_time`, `start_time`, max int, infinity, etc. The exporter shouldn't get any non-ended span so it seems like the `end_time` is being set to some astronomical value somehow 🤔

The raw data for those spans are:

`db.get_partial_state_events`
```jsonc
{
    "traceID": "563478e5d75db03e9fc028822fde0649",
    "spanID": "5b63b0172763c5cf",
    "flags": 1,
    "operationName": "db.get_partial_state_events",
    "references": [
        {
            "refType": "CHILD_OF",
            "traceID": "563478e5d75db03e9fc028822fde0649",
            "spanID": "10c9f2a2c50e286f"
        }
    ],
    "startTime": 1659994809436162,
    "duration": 18446744073709517944,
    "tags": [
        // ...
    ],
    "logs": [],
    "processID": "p1",
    "warnings": null
}
```

`_process_pulled_event`
```
{
    "traceID": "563478e5d75db03e9fc028822fde0649",
    "spanID": "6e9ee2608b03b542",
    "flags": 1,
    "operationName": "_process_pulled_event",
    "references": [
        {
            "refType": "CHILD_OF",
            "traceID": "563478e5d75db03e9fc028822fde0649",
            "spanID": "af2c9c8d08284134"
        }
    ],
    "startTime": 1659994809432356,
    "duration": 18446744073709532019,
    "tags": [
        // ...
    ],
    "logs": [],
    "processID": "p1",
    "warnings": null
},
```

Trying to work out the math to get there

```
213503982d 8h (displayed in Jaeger UI)
5124095576h
18446744073600s
18446744073600000ms
18446744073600000000us
```

```
_process_pulled_event
1659994809432356        (start_time)
18446744073709532019    (duration)
18448404068518965000    (end_time, derived)

db.get_partial_state_events
1659994809436162        (start_time)
18446744073709517944    (duration)
18448404068518957000    (end_time, derived)
```

[`start_time` and `end_time` are calculated by `time.time_ns()`](https://github.com/open-telemetry/opentelemetry-python/blob/1cae35f5ad3c5beabc067e57a3940d3dfeaac8e3/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L867-L880) ([`_time_ns`](https://github.com/open-telemetry/opentelemetry-python/blob/1cae35f5ad3c5beabc067e57a3940d3dfeaac8e3/opentelemetry-api/src/opentelemetry/util/_time.py#L34))

And the Jaeger exporter calculates `startTime` and `duration` with this small [`_nsec_to_usec_round`](https://github.com/open-telemetry/opentelemetry-python/blob/1cae35f5ad3c5beabc067e57a3940d3dfeaac8e3/exporter/opentelemetry-exporter-jaeger-thrift/src/opentelemetry/exporter/jaeger/thrift/translate/__init__.py#L164-L165) function.

Reply:
Definitely close to the Python max int size (unsigned word, 64 bits): `sys.maxsize * 2 + 1` -> `18446744073709551615` (via https://stackoverflow.com/questions/7604966/maximum-and-minimum-values-for-ints)

Reply:
Conversation moved to https://github.com/matrix-org/synapse/pull/13400#discussion_r1199505260